config/initializers/content_security_policy.rb

```diff
@@ -0,0 +1,11 @@
+Rails.application.config.content_security_policy do |p|
+  p.default_src :self, :https
```

Comment:
I think you're also supposed to set `connect_src`? I don`t actually know.

Reply:
It's interesting, this is probably one of the more important security concerns for a site which serves up content designed for other servers...

Reply:
suppose I should get the `connect_src` too ... This is mostly copy pasta from the Rails PR that put this feature into 5.2 with some light reading of the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy) on the header.